tests/checkpoint/test_checkpoint.py

```diff
@@ -419,16 +427,20 @@ class TestCheckpointResult:
     def mock_suite(self, mocker: MockerFixture):
         suite = mocker.Mock(spec=ExpectationSuite)
         suite.name = self.suite_name
+        suite.is_added.return_value = (True, [])
         return suite
 
     @pytest.fixture
     def mock_batch_def(self, mocker: MockerFixture):
-        return mocker.Mock(spec=BatchDefinition)
+        bd = mocker.Mock(spec=BatchDefinition)
+        bd._copy_and_set_values().is_added.return_value = (True, [])
```

Comment:
Unfortunately need to mock this in this manner due to Pydantic - not sure how to get around it